ci/checks/black_lists.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+# Copyright (c) 2019, NVIDIA CORPORATION.
+##########################################
+# cuML black listed function call Tester #
+##########################################
+
+TARGET_BRANCH="master"
+
+# Checkout master for comparison
+git checkout --quiet $TARGET_BRANCH
+
+# Switch back to tip of PR branch
+git checkout --quiet current-pr-branch
```

Comment:
@jirikraus I believe `current-pr-branch` needs to be passed as an argument to this script?

Reply:
I honestly don't know. I took that from the existing changelog ci script:  https://github.com/rapidsai/cuml/blob/branch-0.7/ci/checks/changelog.sh#L11 
@mike-wendt to comment as he contributed `changelog.sh`

Reply:
ah.. that sounds like CI magic! Let's see what Mike has to say about this.

Reply:
I don't think these checkouts are necessary.

Reply:
As said I just took `ci/checks/changelog.sh` as template. What is needed instead or should I just drop those?